packages/@aws-cdk/aws-kinesisanalytics-flink-alpha/README.md

```diff
@@ -59,7 +59,7 @@ snapshotting, monitoring, and parallelism.
 declare const bucket: s3.Bucket;
 const flinkApp = new flink.Application(this, 'Application', {
   code: flink.ApplicationCode.fromBucket(bucket, 'my-app.jar'),
-  runtime: flink.Runtime.FLINK_1_19,
+  runtime: flink.Runtime.FLINK_1_20,
```

Comment:
Can you update row 83 as well for consistency?

Reply:
Thanks. Updated.